src/frontend/web_application/src/layouts/Page/components/Navigation/components/StickyNavBar/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+class StickyNavbarClass extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    stickyClassName: PropTypes.string,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSticky: false,
+    };
+    this.handleScroll = this.handleScroll.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.handleScroll);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.handleScroll);
+  }
+
+  handleScroll(ev) {
+    const scrollSize = ev.srcElement.documentElement.scrollTop || window.scrollY;
+
+    this.setState({ isSticky: scrollSize > 10 });
```

Comment:
You might want to debounce these scroll events, as you're effectively triggering reflow + repaints + react setState + react render on every scroll event